.gitignore

```diff
@@ -194,4 +195,6 @@ coverage_py.xml
 
 .idea/*
 
-python/examples/example_steadystate/model_steadystate_scaled/*
\ No newline at end of file
+python/examples/example_steadystate/model_steadystate_scaled/*
+
+python/sdist/*
```

Comment:
Let's do that more specific. There are files we want to track in this directory.